lang/python/python-texttable/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-texttable
+PKG_VERSION:=1.6.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=texttable
+PKG_HASH:=eff3703781fbc7750125f50e10f001195174f13825a92a45e9403037d539b4f4
+
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-texttable
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Module for creating simple ASCII tables
+  URL:=https://github.com/foutaise/texttable/
+  DEPENDS:=+python3-light
```

Comment:
I think you will also need python3-codecs (for [unicodedata](https://github.com/foutaise/texttable/blob/v1.6.2/texttable.py#L94)).